### PR TITLE
Circle CI: Test shared lib of Hera

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by axic** [comment]
@hugo-dc please let us know here once the tests do not use debug symbols anymore, so the tests here can be retriggered and merged

**Comment by chfast** [comment]
All green!

**Comment by chfast** [comment]
@axic @cdetrio Can I merge this?

File: circle.yml [hunk 1]
```diff
@@ -108,14 +113,25 @@ defaults:
...
+      name: "Install cpp-ethereum"
+      command: |
+        URL=https://github.com/ethereum/cpp-ethereum/releases/download/v1.4.0.dev0/cpp-ethereum-1.4.0.dev0-linux.tar.gz
+        curl -L $URL | sudo tar xz -C /usr/local
+
+  fetch-tests: &fetch-tests
+    run:
+      name: "Fetch eWASM tests"
+      command: |
+        git clone https://github.com/ewasm/tests -b wasm-tests --single-branch --depth 1
```
[the remark this answers is not in the document]
Comment (chfast) [reply circle.yml]: This script does not do anything else. What the point of using it?

[the remark this answers is not in the document]
Comment (axic) [reply circle.yml]: The script does git clone and testing, but doesn't support the shared lib.

**Pull request merged**